src/ObjectUtils/PdbFileTest.h

```diff
@@ -55,15 +55,19 @@ TYPED_TEST_P(PdbFileTest, LoadDebugSymbols) {
   {
     const SymbolInfo& symbol = *symbol_infos_by_address[0x18000ef90];
     EXPECT_EQ(symbol.name(), "PrintHelloWorldInternal");
-    EXPECT_EQ(symbol.demangled_name(), "PrintHelloWorldInternal");
+    // TODO(b/219413222): We actually also expect the parameter list (empty in this case), but the
```

Comment:
If you already want to test the new functionality, you could also just add  a non-typed version of the same test that just used the llvm version. Then later make that typed to cover both cases.

Reply:
Ok, looks good. Yeah, functions with actual arguments would be nice to have in the tests.